client/methods/updateMessage.coffee

```diff
@@ -23,6 +23,7 @@ Meteor.methods
 
 		Tracker.nonreactive ->
 
+			message.editBy = Meteor.userId()
```

Comment:
Could be done like that, I just didn't want to break anything else depending on the `message.ets` field. If we're going crazy, we might as well rename `ets` to `timeStamp`.

Should we do it like this for now and refactor later?